blog/2025-11-24-sharded-repodata-improvements.mdx

```diff
@@ -0,0 +1,264 @@
+authors: [travishathaway]
+tags: [conda, performance]
+description: "Using conda-forge with conda is now significantly faster with improvements made to repodata fetching. Learn how we did it and who helped along the way."
+image: img/blog/2025-11-24-sharded-repodata-improvements/banner.jpg
+---
+import Plot from '@site/src/components/Plot';
+
+<div style={{color: "#555"}}>
+<em>Photo from <a href="https://unsplash.com/de/@scar_tissue?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">
+  Scar Tissue
+</a> on <a href="https://unsplash.com/de/fotos/zerbrochenes-glas-mit-dunklem-verschwommenem-hintergrund-04jpFiARLmY?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a></em>
+</div>
+
+<br />
+
+How are we doing this? With a new feature called "sharded repodata",
+that improves the overall speed of conda when using the [conda-forge](https://conda-forge.org)
+channel. This month we're announcing the beta release of this feature, and in this post, we'll talk
+about how you can try it out, how it came to be, and most importantly how much faster this makes
+conda.
+
+<!-- truncate -->
+
+## How do I try it out?
+
+If you're using conda-forge with conda and would like to try out this new feature, first update
+`conda-libmamba-solver` in your `base` environment and then opt in to the feature by setting
+`plugins.use_sharded_repodata = true`:
+
+```bash
+conda install --name base 'conda-libmamba-solver>=25.11.0'
+conda config --set plugins.use_sharded_repodata true
+```
+## Background and context
+
+In this section, we provide some context around why we decided to develop this feature. We
+think it'll help you understand the performance metrics better, but if you already know
+the ins and outs of how conda stores and uses metadata about its packages, feel free to skip
+ahead to the <a href="#performance-improvements">Performance improvements</a> section.
+
+### What's repodata?
+
+Repodata is a conda specific term that refers to the package index that all conda clients must
+download in order to find and install available packages. The best way to think about it is as
+small database containing all package metadata for a particular channel.
+
+### What's wrong with how it currently works?
+
+Historically, all of the package metadata has been stored in a single file, and when the channel is
+relatively small in size (e.g. Anaconda's "main" channel) this works pretty well. But, for larger
+channels like conda-forge, this has meant longer and longer download times for users as this
+channel has continued to grow over the years.
+
+### How was this fixed?
+
+The sluggishness of downloading large repodata files particularly annoyed the developers
+of the [pixi](https://pixi.sh) package manager who were highly motivated to create a fast experience
+for their users. That's why they led the drafting and development of an entirely new method for
+downloading repodata: sharded repodata.
```

Comment:
Given that the sharded repodata proposal came as a response to the JLAP propsoal, I think this is factual incorrect. It wasn't just the pixi developers, which we should name as Prefix.